vscode-wpilib/resources/webviews/help.html

```diff
@@ -2,31 +2,90 @@
 <html>
   <head>
     <title>WPILib Help</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- CSS will be injected here by WebViewBase's loadWebpage method -->
   </head>
 
   <body>
-    <h1>WPILib Help</h1>
+    <div class="project-container">
+      <img src="replaceresource/resources/wpilib-generic.svg" height="75" />
 
-    <img src="replaceresource/resources/wpilib-generic.svg" height="75" />
+      <h1 class="project-title">WPILib Help</h1>
 
-    <p>Welcome to WPILib Help!</p>
+      <div class="project-row">
+        <p>
+          Welcome to WPILib Help! Thank you for alpha testing the Systemcore and 2027 WPILib. Please
+          see
+          <a href="https://github.com/wpilibsuite/SystemcoreTesting/">
+            https://github.com/wpilibsuite/SystemcoreTesting/</a
+          >
+          for the expectations for testing and to provide your feedback.
+        </p>
 
-    <p>
-      Thank you for alpha testing the Systemcore and 2027 WPILib. Please see
-      <a href="https://github.com/wpilibsuite/SystemcoreTesting/">
-        https://github.com/wpilibsuite/SystemcoreTesting/</a
-      >
-      for the expectations for testing and to provide your feedback.
-    </p>
+        <p>
+          To open this window again, open the Command Palette (F1 or View->Command Palette), and
+          select
+          <b>Open WPILib Help.</b>
+        </p>
+        <p>
+          For more help, see
+          <a href="https://docs.wpilib.org/en/2027">the 2027 WPILib documentation.</a>
+        </p>
+      </div>
 
-    <p>
-      To open this window again, open the Command Palette (F1 or View->Command Palette), and select
-      <b>Open WPILib Help.</b>
-    </p>
+      <div class="project-row">
+        <h2 class="project-subtitle">Common Tasks</h2>
+        <ul>
+          <li>
+            Create a new project: Click on the Command Palette and type "WPILib: Create a new
+            project"
+          </li>
+          <li>
+            Deploy code to the robot: Click on the Command Palette and type "WPILib: Deploy Robot
+            Code"
+          </li>
+          <li>
+            Start a simulation: Click on the Command Palette and type "WPILib: Simulate Robot Code"
+          </li>
+          <li>Install vendor libraries: Click on "WPILib Dependencies" in the Activity Bar</li>
+        </ul>
+      </div>
 
-    <p>
-      For more help, see
-      <a href="https://docs.wpilib.org/en/2027">the 2027 WPILib documentation.</a>
-    </p>
+      <div class="project-row">
+        <h2 class="project-subtitle">Setting Up Your Workspace</h2>
+        <ul>
+          <li>
+            Set your team number: Click on the Command Palette and type "WPILib: Set Team Number"
```

Comment:
setting team number manually isn't a very common task, I think it can be deleted from here